README.md

```diff
@@ -154,6 +154,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Janaki Tech](https://janakitech.com/) | Harihar Bhawan, Lalitpur | We strive to build better software useful for everyone
 - [JankariTech](https://www.jankaritech.com/) | Nayagaun, Pokhara | We specialize in helping customers set up test automation
 - [Javra](https://javra.com/) | Kumaripati, Lalitpur | IT solutions that boost productivity in business
+- [JyabaTech](https://jyaba.tech/) | Shanti Marg, Pokhara | Data Scraping (acquisition) at scale | Data Engineering
```

Comment:
@taragurung Thanks for contributing.

The format of the description is incorrect. Can you remove the additional pipe | and add it as a single line of description?

Reply:
Done! @mesaugat